tensorflow_io/core/kernels/avro/parse_avro_kernels.cc

```diff
@@ -401,6 +421,11 @@ class ParseAvroOp : public OpKernel {
           dense_shapes_[d].dims() > 1 && dense_shapes_[d].dim_size(0) == -1;
     }
 
+    // Check that avro_num_minibatches is positive
```

Comment:
nit: Check that avro_num_minibatches is **non-negative**

Reply:
Updated